### PR TITLE
{bio,lang}[GCCcore/12.2.0,foss/2022b] Ruby v3.0.5, GEMMA v0.98.5

### PR DESCRIPTION
**Pull request by eylenth**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by eylenth** [comment]
Test report by @eylenth
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
by0owl.eu.seeds.basf.net - Linux RHEL 7.9, x86_64, Intel(R) Xeon(R) CPU E5-2620 v4 @ 2.10GHz (broadwell), Python 3.6.8
See https://gist.github.com/eylenth/08257c8d9d91128bc508bd9519e96c44 for a full test report.

File: easybuild/easyconfigs/r/Ruby/Ruby-3.0.5-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,163 @@
+name = 'Ruby'
+version = '3.0.5'
```
Comment (Micket) [inline easybuild/easyconfigs/r/Ruby/Ruby-3.0.5-GCCcore-12.2.0.eb:2]: This version should be bumped up

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/2f3af2313aacd1615b2c3b0511425589 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @eylenth!

**Pull request merged**